android/sentry.properties

```diff
@@ -0,0 +1,3 @@
+defaults.url=https://sentry.frogpond.tech/
+defaults.org=frog-pond-labs
+defaults.project=all-about-olaf
```

Comment:
One thought - could we include a commented out version of the API key line in both of these files?

Reply:
Yeah, we could.